### ISSUE TITLE
White font on black background? Customizable themes?

### ISSUE THREAD

**Issue by alvonellos** [haiwen/seafile-client#796]
Is there any possibility that accessibility features  -- such as customizable themes -- can be added to the seafile client codebase? Particularly white bold font on a blackish background...

**Comment by shoeper** [comment]
Please only submit bugs in GitHub issues (Pro customers should contact us via Email).

Feature requests and installation/usage problem should be asked in the forum https://forum.seafile-server.org/.
